web/rainmaker/dev-packages/egov-ui-kit-dev/src/common/common/ActionMenu/components/index.js

```diff
@@ -332,6 +333,9 @@ class ActionMenuComp extends Component {
                       id={item.name.toUpperCase().replace(/[\s]/g, "-") + "-" + index}
                       onClick={() => {
                         //  localStorageSet("menuPath", item.path);
+                        if (item.name === "Apply TL"){
```

Comment:
Use url instead of display name

Reply:
Also here it will this flag is set for other quick actions we need this feature only for the applt-tl action